x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/suggestions_callout.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { EuiButtonTo } from '../../../../shared/react_router_helpers';
+import { useLocalStorage } from '../../../../shared/use_local_storage';
+import { useLocation } from 'react-router-dom';
+
+interface SuggestionsCalloutProps {
+  title: string;
+  description: string;
+  buttonTo: string;
+  lastUpdatedTimestamp: string; // ISO string like '2021-10-04T18:53:02.784Z'
+}
+
+export const SuggestionsCallout: React.FC<SuggestionsCalloutProps> = ({
+  title,
+  description,
+  buttonTo,
+  lastUpdatedTimestamp,
+}) => {
+  const { pathname } = useLocation();
+
+  const [lastDismissedTimestamp, setLastDismissedTimestamp] = useLocalStorage<string>(
+    `suggestions-callout--${pathname}`,
```

Comment:
some example keys:
`suggestions-callout--/xyz/app/enterprise_search/app_search/engines/some-engine`
`suggestions-callout--/app/enterprise_search/app_search/engines/some-engine/analytics`
`suggestions-callout--/xyz/app/enterprise_search/app_search/engines/some-engine/curations/1-2-3`

Using pathname seems better than requiring an id prop to be passed in.  If we find ourselves using multiple of theses on a page (which we shouldn't be) we can extend this with an id later.